lib/propmangle.js

```diff
@@ -135,38 +135,51 @@ function mangle_properties(ast, options) {
     var names_to_mangle = [];
     var unmangleable = [];
 
-    // step 1: find candidates to mangle
     ast.walk(new TreeWalker(function(node){
+        var keep_quoted_strict = options.keep_quoted === "strict";
```

Comment:
`var keep_quoted_strict` should be moved before the step 1`walk` so it doesn't incur a cost for each node visited. That way the declaration need not be repeated later at line 166.

Also, please don't remove the comment.